uploader/templates/datepicker.html

```diff
@@ -0,0 +1,6 @@
+<div class="form-group row">
+	<label for="dateCreated" class="col-2 col-form-label"></label>
+	<div class="col-10">
+		<input class="form-control" type="date" name="dateCreated" id="dateCreated" onchange="TDate()" required />
```

Comment:
Sorry, but this id wont work. This would just put the same for everyone. I dont know if it would work, but maybe: 
```
id="wpDateCreated-${cell.getData()['id']}"
```
or something like that ? (I just copied this from licenses.html) I might be wrong.